libc/unistd_getopt.c

```diff
@@ -314,6 +313,101 @@ TEST(unistd_getopt, getopt_unknownopt_noarg)
 }
 
 
+TEST(unistd_getopt, getopt_unknownopt_multiple)
+{
+	/* mocks of main() arguments */
+	char *testargv1[] = { "cmd", "-xxx", "", "" };
```

Comment:
no need for empty strings as `testargc = 2`

Reply:
> pointed out some small things which could be improved further. Did not run the test on host.

Thanks, applied.